### ISSUE TITLE
Label placement strategy

### ISSUE THREAD

**Issue by MeetMichal** [beto-rodriguez/LiveCharts2#1357]
**Is your feature request related to a problem? Please describe.**
Labels are distributed automatically across axis and I dont have any control over it. Sometimes labels covers middle of the axis, sometimes  they cover also the min and max.

**Describe the solution you'd like**
I want to have a possibility to decide how labels are distributed over axis. I want to decide whether labels should cover min and max value of the chart. 

On the screen below labels for left axis end with 250000, while max value is closer to 290000 leaving quite a big gap without labels. Also left and right axes look strange together - there are 5 labels on one and 11 on the other
![image](https://github.com/beto-rodriguez/LiveCharts2/assets/150547065/bfba90de-6b7c-447b-8c93-5b5ea6ee044e)

**Comment by beto-rodriguez** [comment]
Hi, you also create your own separators:

https://livecharts.dev/docs/Blazor/2.0.0-rc2/samples.axes.customSeparatorsInterval

is that enough? ideally the library should handle this better, but we have options to build our own implementations.

**Comment by MeetMichal** [comment]
Yeah, I know about separators. I just feel that some flag where user could set "Show me labels for Min and Max" would be great. I guess for now I will go with custom separators :)

**Comment by beto-rodriguez** [comment]
In the next version, this is improved with #1643:


![ii](https://private-user-images.githubusercontent.com/10853349/375981418-a76a31e3-88d3-4a84-93d0-fc36ce04e1df.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk2Mjc3NTAsIm5iZiI6MTcyOTYyNzQ1MCwicGF0aCI6Ii8xMDg1MzM0OS8zNzU5ODE0MTgtYTc2YTMxZTMtODhkMy00YTg0LTkzZDAtZmMzNmNlMDRlMWRmLmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDIyVDIwMDQxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE2NDk5ZDFmMTdkMTQzZDlmYmVmMjMyYzM4MjQ4MmZiYjkzMDg4OGYxMTM4ZWQ2MzE4NjBkYzliMTcyZTU3Y2MmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.EAIHWUBEFeV7TjnkfxeCJYCS4U5mMFWrS7Us9Q1N_TA)

I'll keep this open, so we can get feedback if that option is not enough yet.

**Comment by beto-rodriguez** [comment]
I will close this for now, since we can now control the `LabelsDensity` property

Thanks for the report!
